plugins/secretstores/os/sample_darwin.conf

```diff
@@ -0,0 +1,17 @@
+# Operating System native secret-store
+[[secretstores.os]]
+  ## Unique identifier for the secret-store.
+  ## This id can later be used in plugins to reference the secrets
+  ## in this secret-store via @{<id>:<secret_key>} (mandatory)
+  id = "secretstore"
+
+  ## MacOS' Keychain name and service name
+  # keyring = "telegraf"
+  # collection = ""
+
+  ## MacOS' Keychain password
+  ## If no password is specified here, Telegraf will prompt for it at startup time.
+  # password = ""
+
+  ## Allow dynamic secrets that are updated during runtime of telegraf
+  # dynamic = false
```

Comment:
Are these example files expected to have newlines?

Reply:
Yes. Fixed.